docs/interfaces/autoupdate-service-examples.txt

```diff
@@ -251,11 +259,15 @@ ModelRequest: {
     ids: [2, 4],  # 4 does not exists
     fields: {
         content_object_ids: {
+            type: "generic-relation-list",
             fields: {
                 a: null,
                 b: null,
                 not_existent: {
-                    key: null,
+                    type: "relation-list",
+                    fields: {
+                        key: null,
+                    }
```

Comment:
This is a syntax error. If the type is "relation-list", then the object needs the field "collection". I used `generic-relation` in the golang tests: https://github.com/OpenSlides/openslides-autoupdate-service/pull/43/files#diff-a338522804dc3046ef5f239f5a697f8aR188-R191

So either use `generic-relation` or `generic-relation-list` or add the attribute `collection`